### PR TITLE
Na workspacescreen

### PR DESCRIPTION
**Pull request by nehaagrawalmm**
Project Selection list and some ui Changes on new and existing sign up screens

### CODE REVIEW

File: androidApp/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,6 +13,10 @@
         tools:ignore="RtlEnabled"
         android:usesCleartextTraffic="true"
         tools:targetApi="donut">
+        <activity
+            android:name=".ui.screens.projectScreen.ProjectListActivity"
+            android:exported="false"
+            android:theme="@style/AppTheme" />
```
Comment (shubhamsinghmutualmobile) [inline androidApp/src/main/AndroidManifest.xml:19]: Can we remove this activity?

File: androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/projectScreen/ProjectScreen.kt [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        ) { bodyPadding ->
+
+        Column(modifier = Modifier.padding(bodyPadding)) {
+
+            LazyColumn(modifier = Modifier.fillMaxWidth()) {
+                val searchedText = textState.value.text
+                filteredMap = if (searchedText.isEmpty()) {
+                    listMap
+                } else {
+                    var resultList1 = emptyMap<String, List<String>>()
```
Comment (shubhamsinghmutualmobile) [inline androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/projectScreen/ProjectScreen.kt:99]: Can we try to name this variable better?

File: androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/timeScreen/TimeScreen.kt [hunk 3]
```diff
@@ -53,14 +55,16 @@ const val MaxItemFling = 1
...
     onWeekScrolled: (Int) -> Unit,
     onDayScrolled: (Int) -> Unit,
 ) {
     val ctx = LocalContext.current
     Scaffold(
         floatingActionButton = {
             FloatingActionButton(onClick = {
-                ctx.startActivity(Intent(ctx, NewEntryActivity::class.java))
+                                           navController.navigate(ScreenList.NewEntryScreen())
+               // ctx.startActivity(Intent(ctx, NewEntryActivity::class.java))
```
Comment (shubhamsinghmutualmobile) [inline androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/timeScreen/TimeScreen.kt:67]: Let's remove all the commented code

File: shared/src/commonMain/kotlin/com/mutualmobile/harvestKmp/features/datamodels/authApiDataModels/SignUpDataModel.kt [hunk 4]
```diff
@@ -49,7 +49,7 @@ class SignUpDataModel(private val onDataState: (DataState) -> Unit) :
             when (val signUpResponse = existingOrgSignUpUseCase(
                 firstName = firstName,
                 lastName = lastName,
-                company = company,
+                company = confirmPassword,
```
Comment (shubhamsinghmutualmobile) [inline shared/src/commonMain/kotlin/com/mutualmobile/harvestKmp/features/datamodels/authApiDataModels/SignUpDataModel.kt:52]: Can we rename this variable?

**Review by shubhamsinghmutualmobile — COMMENTED** [review]
@nehaagrawalmm Please check the comments

File: androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/signUpScreen/NewOrgSignUpScreen.kt [hunk 5]
```diff
@@ -31,83 +24,109 @@ import com.mutualmobile.harvestKmp.features.datamodels.authApiDataModels.SignUpD
...
+                    onClick = {
+                        signUpDataModel.signUp(
+                            currentFirstName,
+                            currentLastName,
+                            currentWorkEmail,
+                            currentPassword,
+                            confirmPassword,
+                            companyName,
+                            companyWebsite,
+                            companyIdentifier
```
[the remark this answers is not in the document]
Comment (nehaagrawalmm) [reply androidApp/src/main/java/com/mutualmobile/harvestKmp/android/ui/screens/signUpScreen/NewOrgSignUpScreen.kt]: done

**Review by shubhamsinghmutualmobile — APPROVED** [review]

**Pull request merged**